Source/JavaScriptCore/llint/LowLevelInterpreter.asm

```diff
@@ -2778,6 +2778,10 @@ op(wasm_function_prologue, macro ()
     crash()
 end)
 
+op(wasm_function_prologue_simd_trampoline, macro ()
```

Comment:
For my own elucidation -- how do you know whether to call this trampoline vs. the normal one when you're calling a wasm function? Is there a distinction between wasm functions which use any simd features vs. those which use none?

Reply:
We call the correct trampoline based on if the function uses SIMD, which is set in the FunctionParser. When we set up the Callee object, we'll initialize it with the right prologue.